### PR TITLE
[359] Mailchimp integration & Users subscription

### PR DESCRIPTION
**Pull request by kushniryb**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -30,7 +30,7 @@ gem 'uglifier',                                                       '~> 3.2.0'
 gem 'rails-assets-svgxuse',                   source: 'https://rails-assets.org'
 gem 'rails-assets-lory',                      source: 'https://rails-assets.org'
 
-#=== FEATUES ===================================================================
+#=== FEATURES ===================================================================
```
Comment (sveredyuk) [inline Gemfile]: 👍

File: app/views/users/registrations/edit.html.slim [hunk 2]
```diff
@@ -12,6 +12,8 @@ h2
     .pk-input__container.mtm = f.input :password, placeholder: 'Password', autocomplete: "off", hint: "leave it blank if you don't want to change it", required: false, label_html: {class: 'pk-input__label'}, input_html: {class: 'pk-input'}
     .pk-input__container.mts = f.input :password_confirmation, placeholder: 'Password comfirmation', required: false, label_html: {class: 'pk-input__label'}, input_html: {class: 'pk-input'}
     .pk-input__container.mtm = f.input :current_password, placeholder: 'Current password', hint: "we need your current password to confirm your changes", required: true, label_html: {class: 'pk-input__label'}, input_html: {class: 'pk-input'}
+    .pk-input__container.mtm = f.input :subscribed, as: :boolean, label: 'Subscribe Me'
```
Comment (sveredyuk) [inline app/views/users/registrations/edit.html.slim]: Subscribe Me to Pivorak's newsletter

File: app/views/users/registrations/new.html.slim [hunk 3]
```diff
@@ -13,6 +13,7 @@ h2
       input_html: {class: 'pk-input'}
     .pk-input__container.mts.mbx = f.input :password_confirmation, required: true, placeholder: 'Password confirmation', label_html: {class: 'pk-input__label'},
       input_html: {class: 'pk-input'}
+    .pk-input__container.mts.mbx = f.input :subscribed, as: :boolean, label: 'Subscribe Me'
```
Comment (sveredyuk) [inline app/views/users/registrations/new.html.slim]: same

File: app/workers/mailchimp/subscription_worker.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+module Mailchimp
+  class SubscriptionWorker
+    include Sidekiq::Worker
```
Comment (sveredyuk) [inline app/workers/mailchimp/subscription_worker.rb]: Please, use ActiveJob adapter

**Review by sveredyuk — APPROVED** [review]

**Comment by kushniryb** [comment]
@sveredyuk I'll probably add some tests to improve the coverage so do not merge it yet

**Comment by kushniryb** [comment]
@sveredyuk Don't forget to add `MAILCHIMP_API_KEY` & `MAILCHIMP_LIST_ID` to `.env`

**Review by sveredyuk — APPROVED** [review]

**Pull request merged**